### ISSUE TITLE
Sense color advantages

### ISSUE THREAD

**Issue by SiriusAshling** [sparkle-preference/OriDERandomizer#36]
Adjusting the Hot/Cold colors for sense can give notable advantages.
This feature is important for accessibility and valuable for personal preference, but with a competitive mindset there is actually no choice on what to use, black to white outperforms any other choice.
This is because at the outer ends of teh sense range, the color change is very faint, and black to white allows you to spot the contrast with Ori's (always black) eyes at distances where other color choices would not allow most humans to spot anything.
It would be great if sense was designed so that the hot/cold colors could actually be chosen by preference without having to consider competitive advantages.
A possible solution to this would be to have an instant shift of color as soon as something enters sense range (e.g. immediately change 25% of the way) and then resume the gradient behaviour based on distance as currently. This would equalize different color choices (by making everything as strong as black to white currently is).
